landing_page/app/models/user.rb

```diff
@@ -21,6 +21,6 @@ def User.digest(string)
     BCrypt::Password.create(string, cost: cost)
   end
 
-  has_many :microposts, dependent: :destroy
```

Comment:
Why did you remove this?

Reply:
I didn't mean to delete it so thanks for spotting it 👍